.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+env:
+  - PYTHON_VERSION=2.7
+before_install:
+  - wget http://repo.continuum.io/miniconda/Miniconda3-3.7.3-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda3/bin:$PATH
+  # Update conda itself
+  - conda update --yes conda
+install:
+  - conda create --yes -n env_name python=$PYTHON_VERSION pip nose numpy scipy matplotlib pandas>=0.15 flake8
+  - conda install --yes -n env_name python=$PYTHON_VERSION h5py>=2.3.1 statsmodels lxml
```

Comment:
one is `conda create` and the other is `conda install`; I don't think `create` will work if I put statsmodels and lxml on that line, will it?